uvicorn/protocols/websocket.py

```diff
@@ -82,8 +82,8 @@ def put_message(self, message):
 
         if self.protocol.state == websockets.protocol.State.OPEN:
             if not self.protocol.accepted:
-                accept = bool(message_type == 'websocket.accept')
-                close = bool(message_type == 'websocket.close')
+                accept = not not message_type == 'websocket.accept'
+                close = not not message_type == 'websocket.close'
```

Comment:
`not not message_type`

🤔

Reply:
`close = (message_type == 'websocket.close')` ?

Reply:
`not not` sounds like a JavaScript.

```python
close = message_type == 'websocket.close'
```

This is totally enough and there is no need for parenthesis.

```python
>>> message_type = 'websocket.close'
>>> close = message_type == 'websocket.close'
>>> close
True
```

Reply:
Yes, tho possibly more readable with the explicit parenthesis. It's a bit hard to otherwise tell if it's intended to be like that otherwise, or an error.

Reply:
It's python, explicit parenthesis are not needed, for example, no one writes:

```python
if (condition):
    ...
```

But I guess need for explicit parenthesis comes from experience with other programming languages where parenthesis are required.

Nevermind.

Reply:
I had initially just went with `not not` because I was replacing the `bool` call based on the information on this page https://wiki.python.org/moin/PythonSpeed, but it doesn't really matter here.